frappe/core/doctype/data_import/data_import.js

```diff
@@ -84,6 +84,8 @@ frappe.ui.form.on('Data Import', {
 			frm.doc.docstatus === 0 && (!frm.doc.import_status || frm.doc.import_status == "Failed")) {
 			frm.page.set_primary_action(__("Start Import"), function() {
 				frappe.call({
+					freeze: true,
```

Comment:
Freezing the screen is not a good pattern. Please remove `freeze: true`. Passing the `btn` will ensure that it is not clicked twice.